diff --git a/arch/arm64/boot/dts/qcom/sc7180.dtsi b/arch/arm64/boot/dts/qcom/sc7180.dtsi
index b5ebf8980..01f258e27 100644
--- a/arch/arm64/boot/dts/qcom/sc7180.dtsi
+++ b/arch/arm64/boot/dts/qcom/sc7180.dtsi
@@ -2901,90 +2901,6 @@ usb_1_qmpphy: phy@88e8000 {
 			#phy-cells = <1>;
 		};
 
-		pmu@90b6300 {
-			compatible = "qcom,sc7180-cpu-bwmon", "qcom,sdm845-bwmon";
-			reg = <0 0x090b6300 0 0x600>;
-			interrupts = <GIC_SPI 581 IRQ_TYPE_LEVEL_HIGH>;
-
-			interconnects = <&gem_noc MASTER_APPSS_PROC QCOM_ICC_TAG_ACTIVE_ONLY
-					 &gem_noc SLAVE_LLCC QCOM_ICC_TAG_ACTIVE_ONLY>;
-			operating-points-v2 = <&cpu_bwmon_opp_table>;
-
-			cpu_bwmon_opp_table: opp-table {
-				compatible = "operating-points-v2";
-
-				opp-0 {
-					opp-peak-kBps = <2288000>;
-				};
-
-				opp-1 {
-					opp-peak-kBps = <4577000>;
-				};
-
-				opp-2 {
-					opp-peak-kBps = <7110000>;
-				};
-
-				opp-3 {
-					opp-peak-kBps = <9155000>;
-				};
-
-				opp-4 {
-					opp-peak-kBps = <12298000>;
-				};
-
-				opp-5 {
-					opp-peak-kBps = <14236000>;
-				};
-			};
-		};
-
-		pmu@90cd000 {
-			compatible = "qcom,sc7180-llcc-bwmon", "qcom,sc7280-llcc-bwmon";
-			reg = <0 0x090cd000 0 0x1000>;
-			interrupts = <GIC_SPI 241 IRQ_TYPE_LEVEL_HIGH>;
-
-			interconnects = <&mc_virt MASTER_LLCC QCOM_ICC_TAG_ACTIVE_ONLY
-					 &mc_virt SLAVE_EBI1 QCOM_ICC_TAG_ACTIVE_ONLY>;
-			operating-points-v2 = <&llcc_bwmon_opp_table>;
-
-			llcc_bwmon_opp_table: opp-table {
-				compatible = "operating-points-v2";
-
-				opp-0 {
-					opp-peak-kBps = <1144000>;
-				};
-
-				opp-1 {
-					opp-peak-kBps = <1720000>;
-				};
-
-				opp-2 {
-					opp-peak-kBps = <2086000>;
-				};
-
-				opp-3 {
-					opp-peak-kBps = <2929000>;
-				};
-
-				opp-4 {
-					opp-peak-kBps = <3879000>;
-				};
-
-				opp-5 {
-					opp-peak-kBps = <5931000>;
-				};
-
-				opp-6 {
-					opp-peak-kBps = <6881000>;
-				};
-
-				opp-7 {
-					opp-peak-kBps = <8137000>;
-				};
-			};
-		};
-
 		dc_noc: interconnect@9160000 {
 			compatible = "qcom,sc7180-dc-noc";
 			reg = <0 0x09160000 0 0x03200>;
